graze-cli/GRAZE.md

```
@@ -0,0 +1,74 @@
+- India bootcamp / global south optimized (free-tier providers, Colab secrets)
+
+Copy this `graze-cli/` directory to your project root. Graze auto-detects it.
+
+## Installation (Open Paws fork)
+
+**NOT** `npm i -g opencode` (upstream, lacks advocacy rules).
+
+### Global install (recommended for Guild devs)
+
+```bash
+git clone https://github.com/Open-Paws/graze-cli.git
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)